### DISCUSSION TITLE
SimpleSDXL更新：优化国内模型下载，可自制预置包，启用MUID模型文件标识，可动态切换和自动下载

### DISCUSSION THREAD

**Discussion by metercai** [lllyasviel/Fooocus#1623] [Show and tell]
Fooocus中文孪生版，优化了增强的相册图集、顶部导航和右侧工具箱。预置包生成流程和加载流程优化，启用MUID模型文件标识，文件名不同自动转换，本地无模型文件自动下载，自动识别embeddings模型。下载器针对国内网络进行优化，包索引库、项目代码库和模型库等全都选用国内源进行更新和下载。一键启动器瘦身到75M，与Fooocus同环境运行的升级包40M。适配多个log.html版本，保持全兼容，同步升级。https://github.com/metercai/SimpleSDXL
![SimpleSDXL](https://github.com/lllyasviel/Fooocus/assets/5652458/32e8f6b5-5bb2-4a69-8932-49101f6f03a6)
品尝一下中国的饺子😄

**Comment by siriume** [comment]
I think #1570 may be better when switching presets, because it will not switch the presets when generating images, and the page will not return to the output.

**Reply by metercai** [reply]
emm, session based is lighter and more independent.  If switching is not frequent, the current mode is ok, implementing simple and style controllable. it can be replaced with session based after have time.

**Reply by metercai** [reply]
@siriume  Upgraded to switching preset session based. It can avoid many problems.

**Comment by grady-lee** [comment]
There are many built-in modes that can be easily switched, lowering the threshold for people to use, and it becomes very simple for novices to use, even if they don't know the program, they can also realize image processing very conveniently. This is the software I need
